### PR TITLE
DOCS-4030-Jira-tickets-private-attachment-section

### PR DESCRIPTION
**Pull request by BonnieF**
Updating the public and Private upload areas for files.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/Contributing/IssueReporting/_index.md [hunk 1]
```diff
@@ -15,20 +15,19 @@ tags:
...
 
-## Bug Reports
+### Bug Reports
 
 Go to the [TrueNAS project on Jira](https://ixsystems.atlassian.net/jira/software/c/projects/NAS/issues) and click **Create** in the top bar.
 
 ![Create Ticket](/images/Contribute/JiraCreate.png "Create Ticket")
 
-Set the *Issue Type* to *Bug*. The form will reload with more fields.
+Set the *Issue Type* to **Bug**. The form reloads with more fields.
```
Comment (micjohnson777) [inline content/Contributing/IssueReporting/_index.md:30]:
```suggestion
Set the **Issue Type** to **Bug**. The form reloads with more fields.
```

File: content/Contributing/IssueReporting/_index.md [hunk 2]
```diff
@@ -15,20 +15,19 @@ tags:
...
 Developers use most of them, but you should fill out the *Summary* and *Description* to provide a useable report.
```
Comment (micjohnson777) [inline content/Contributing/IssueReporting/_index.md:31]: Also fix line 33 and 35.
```suggestion
Developers use most of them, but you should fill out the **Summary** and **Description** to provide a useable report.
```

File: content/Contributing/IssueReporting/_index.md [hunk 3]
```diff
@@ -60,63 +59,55 @@ A good bug report includes these elements:
 
 * Include additional details you think can help the developer investigate.
 
-When finished filling out the *Description*, click *Create* at the bottom of the form.
+When finished filling out the *Description*, click **Create** at the bottom of the form.
```
Comment (micjohnson777) [inline content/Contributing/IssueReporting/_index.md:62]:
```suggestion
When finished filling out the **Description**, click **Create** at the bottom of the form.
```

**Review by micjohnson777 — COMMENTED** [review]
Excellent conversion from tabs to expands!  Just a few things found. Also, if there are new images, suggest putting them in a 22.12 folder to keep them separate from the older scale release and let others know these are related to the Bluefin release.

File: content/Contributing/IssueReporting/_index.md (hunk 2 above)
Comment (BonnieF) [reply content/Contributing/IssueReporting/_index.md]: Fixed.

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**